genesis/process/intermediate/deployLibrarySC.go

```diff
@@ -95,8 +93,12 @@ func (dp *deployLibrarySC) Deploy(sc genesis.InitialSmartContractHandler) error
 		return err
 	}
 
+	isForCurrentShard := func(address []byte) bool {
```

Comment:
this could have been a non-anonymous function, also, might have signaled this as a handler: isForCurrentShardFn